perception/lidar_transfusion/include/lidar_transfusion/lidar_transfusion_node.hpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIDAR_TRANSFUSION__LIDAR_TRANSFUSION_NODE_HPP_
+#define LIDAR_TRANSFUSION__LIDAR_TRANSFUSION_NODE_HPP_
+
+#include "lidar_transfusion/detection_class_remapper.hpp"
+#include "lidar_transfusion/postprocess/non_maximum_suppression.hpp"
+#include "lidar_transfusion/preprocess/pointcloud_densification.hpp"
+#include "lidar_transfusion/transfusion_trt.hpp"
+#include "lidar_transfusion/visibility_control.hpp"
+
+#include <rclcpp/rclcpp.hpp>
+#include <tier4_autoware_utils/ros/debug_publisher.hpp>
+#include <tier4_autoware_utils/ros/published_time_publisher.hpp>
+#include <tier4_autoware_utils/system/stop_watch.hpp>
+
+#include <autoware_auto_perception_msgs/msg/detected_object_kinematics.hpp>
+#include <autoware_auto_perception_msgs/msg/detected_objects.hpp>
+#include <autoware_auto_perception_msgs/msg/object_classification.hpp>
+#include <autoware_auto_perception_msgs/msg/shape.hpp>
+#include <sensor_msgs/msg/point_cloud2.hpp>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace lidar_transfusion
+{
+
+class LIDAR_TRANSFUSION_PUBLIC LidarTransfusionNode : public rclcpp::Node
+{
+public:
+  explicit LidarTransfusionNode(const rclcpp::NodeOptions & options);
+  void cloudCallback(const sensor_msgs::msg::PointCloud2::ConstSharedPtr msg);
+
+private:
+  rclcpp::Subscription<sensor_msgs::msg::PointCloud2>::ConstSharedPtr cloud_sub_;
+  rclcpp::Publisher<autoware_auto_perception_msgs::msg::DetectedObjects>::SharedPtr objects_pub_;
+
+  tf2_ros::Buffer tf_buffer_;
+  tf2_ros::TransformListener tf_listener_{tf_buffer_};
+
+  DetectionClassRemapper detection_class_remapper_;
+  float score_threshold_{0.0};
+  std::vector<std::string> class_names_;
+
+  NonMaximumSuppression iou_bev_nms_;
+
+  std::unique_ptr<TransfusionTRT> detector_ptr_;
```

Comment:
nit: pointers should use uniform initialization with a nullptr

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/6890/commits/be8914182ee63fc1962e68927fb11b7e88c85b1c